### ISSUE TITLE
pydrake: Installed version can only use `GetDrakePath` once before it fails?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [RobotLocomotion/drake#7624]
When testing using the installed Python #7323, I encountered an issue where calling `GetDrakePath` more than once causes it to fail:
```
$ cd /home/eacousineau/proj/tri/repo/branches/drake-distro/master
$ mkdir -p build_install
$ bazel --bazelrc=/dev/null run //:install -- ~+/build_install
$ export PYTHONPATH=~+/build_install/lib/python2.7/site-packages:${PYTHONPATH}
$ python -c 'import pydrake; print(pydrake.getDrakePath()); print(pydrake.getDrakePath())'
/home/eacousineau/proj/tri/repo/branches/drake-distro/master/build_install/share/drake/drake
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/eacousineau/proj/tri/repo/branches/drake-distro/master/build_install/lib/python2.7/site-packages/pydrake/__init__.py", line 14, in getDrakePath
    return abspath(common.GetDrakePath())
RuntimeError: could not find resource: drake/.drake-find_resource-sentinel
```

Possibly relates #7593?

\cc @jwnimmer-tri

**Comment by jwnimmer-tri** [comment]
@fbudin69500 Would it make sense for you to look into this?

**Comment by fbudin69500** [comment]
Sorry, I hadn't seen this issue until now. I'll look into it.

**Comment by EricCousineau-TRI** [comment]
Awesome, thanks!

**Comment by EricCousineau-TRI** [comment]
Closed via #7711.
